desktop/src/main.rs

```diff
@@ -27,7 +27,7 @@ fn main() {
 		// We are in a CEF subprocess
 		// This will block until the CEF subprocess quits
 		let error = cef_context_builder.execute_sub_process();
-		tracing::error!("Cef subprocess failed with error: {error}");
```

Comment:
Was this included by accident or do we want this to be a debug!?

Reply:
Hmm I think I'd prefer this to be a warning then. As per convention we use `debug!` calls for debugging but basically never ship code with debug logs (except for very few exceptions). Another change I would like to see at some point is that we set the log level for packages we build ourselves to debug when not compiling in release mode so this would break anyway once we do that